website/docs/r/application_gateway.html.markdown

```diff
@@ -41,7 +41,8 @@ resource "azurerm_public_ip" "example" {
   name                = "example-pip"
   resource_group_name = azurerm_resource_group.example.name
   location            = azurerm_resource_group.example.location
-  allocation_method   = "Dynamic"
+  allocation_method   = "Static"
+  sku                 = "Standard"
```

Comment:
Since this has become the default value for public IP in v4.0+ this doesn't need to be specified in the config 
```suggestion
```